tests/playground.ts

```diff
@@ -0,0 +1,150 @@
+import { LoginAsRootPage } from "../pom/login-as-root-page.js";
+import { ProductSelectionPage } from "../pom/product-selection-page.js";
+import fs from "fs";
+import path from "path";
+
+import puppeteer, { ElementHandle } from "puppeteer-core";
+import { expect } from "chai";
```

Comment:
I would recommend you now that you mentioned to me offline that works this preliminary version, to make the changes in the real file, not in playground.ts, because I noticed that you are changing here build/tests/test_install.js{.map,} and doesn't make sense, otherwise set to WIP please.